modules/howtos/pages/managing-connections.adoc

```diff
@@ -209,11 +209,34 @@ NOTE: Any TLS certificates must be set up at the point where the connections are
 
 WARNING: If the client cannot load or was not built with OpenSSL, attempting an SSL connection will result in a 'FEATURE_UNAVAILABLE'.
 
-Couchbase Server Enterprise Edition supports full encryption of client-side traffic using Transport Layer Security (TLS).
+Couchbase Server Enterprise Edition and Couchbase Capella support full encryption of client-side traffic using Transport Layer Security (TLS).
 That includes key-value type operations, queries, and configuration communication.
-Make sure you have the Enterprise Edition of Couchbase Server before proceeding with configuring encryption on the client side.
+Make sure you have the Enterprise Edition of Couchbase Server, or a Couchbase Capella account, before proceeding with configuring encryption on the client side.
 
-To configure encryption for the Python SDK:
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+The Python SDK bundles Capella’s standard root certificate by default.
+This means you don’t need any additional configuration to enable TLS -- simply use `couchbases://` in your connection string.
+
+NOTE: Capella's root certificate is *not* signed by a well known CA (Certificate Authority).
+However, as the certificate is bundled with the SDK, it is trusted by default.
+--
+
+Couchbase Server::
++
+--
+As of SDK 4.1, if you connect to a Couchbase Server cluster with a root certificate issued by a trusted CA (Certificate Authority), you no longer need to configure the `cert_path` property in the `PasswordAuthenticator`.
+
+The cluster's root certificate just needs to be issued by a CA whose certificate is in your system trust store.
+This includes publicly trusted CAs (e.g., GoDaddy, Verisign, etc...), plus any other CA certificates that you wish to add.
+The Python SDK uses https://github.com/couchbaselabs/couchbase-cxx-client[{cbpp}] internally to retrieve the platform root CA.
+
+IMPORTANT: {cbpp} loads your platform's root CA store using OpenSSL, therefore you should make sure that your system or platform is configured to support this.
+
+You can still provide a certificate explicitly if necessary:
```

Comment:
@thejcfactor, when you have a minute could you look this over, not sure if there are any other caveats to add for Python?